sass/base/components/just-in-time-block/_just-in-time.scss

```diff
@@ -7,10 +6,18 @@
 
 .form__row--just-in-time-block {
 
-  .toggle-link {
+  .link.toggle-link {
```

Comment:
using this means that donate  will use that grey colour - would it not be better to let .link determine the link colour in the theme which I think some are already doing

Reply:
Okeydokey, I'll update this in a new PR now